src/renderers/webgl/WebGLObjects.js

```diff
@@ -190,7 +190,10 @@ function WebGLObjects( gl, properties, info ) {
 
 		var property = properties.get( geometry );
 
-		if ( property.wireframe !== undefined ) {
+		var numberOfPoints = geometry.attributes.position.array/3;
+
+		if ( property.wireframe !== undefined &&
+		   property.wireframe.count/2 == numberOfPoints ) {
```

Comment:
I do not expect this will work in general, as both indexed and non-indexed geometries would have to be supported.